packages/autorest.python/autorest/codegen/templates/model_base.py.jinja2

```diff
@@ -570,6 +570,36 @@ class Model(_MyMutableMapping):
             result[k] = Model._as_dict_value(v, exclude_readonly=exclude_readonly)
         return result
 
+{% if code_model.has_form_data %}
+    def _as_origin_dict(self, *, exclude_readonly: bool = False) -> typing.Dict[str, typing.Any]:
+        """Return a dict that the value is the origin value instead of serialized value.
+
+        :keyword bool exclude_readonly: Whether to remove the readonly properties.
+        :returns: A dict JSON compatible object
+        :rtype: dict
+        """
+
+        result = {}
+        for attr, field in self._attr_to_rest_field.items():
+            if exclude_readonly and _is_readonly(rest_field):  # pyright: ignore[reportUnboundVariable]
+                continue
+            rest_name = field._rest_name  # pylint: disable=protected-access
+            value = self.get(rest_name)
+            if value is not None:
+                result[rest_name] = Model._as_origin_dict_value(getattr(self, attr), exclude_readonly=exclude_readonly)
+        return result
```

Comment:
If only permit input `JSON` type, we can avoid the method but I don't think it is good solution.

Reply:
I mean you could add a more specific method for multi-part form data instead of a common `as_raw_dict` method which could make the logic cleaner. I have no idea about the current `as_raw_dict` could be used in another scenario,

Reply:
done